src/AddIdea.tsx

```diff
@@ -183,11 +181,12 @@ export default function AddIdea({
   return (
     <>
       <div
-        className={`p-8 absolute bottom-0 left-0 w-full z-50 transition-all duration-300 ease-in-out ${
+        className={cn(
```

Comment:
I don't understand the use of `cn()`, is it due to the shadcn librairie ?

Reply:
It has nothing to do with shadcn, the cn is there to avoid making
It's a package like [classnames](https://www.npmjs.com/package/classnames) or [clsx](https://www.npmjs.com/package/clsx), which makes it easier to group your conditions in your className, which can quickly become a mess.
```
className={`flex bg-red-500 ... ${isAddIdeaOpen ? translate-y-0 : 'pointer-events-none translate-y-full'}`}`}
```